iceoryx_hoofs/include/iceoryx_hoofs/cxx/span.hpp

```diff
@@ -0,0 +1,414 @@
+// Copyright (c) 2021 - 2022 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+#ifndef IOX_HOOFS_CXX_SPAN_HPP
+#define IOX_HOOFS_CXX_SPAN_HPP
+
+#include "iceoryx_hoofs/cxx/requires.hpp"
+#include "iceoryx_hoofs/cxx/span_iterator.hpp"
+#include "iceoryx_hoofs/cxx/type_traits.hpp"
+
+#include <algorithm>
+#include <array>
+#include <cstddef>
+#include <cstdint>
+#include <cstdio>
+#include <limits>
+#include <type_traits>
```

Comment:
The trampoline already includes this here, so an additional include is not needed.

```suggestion
```